python/sparkdl/graph/utils.py

```diff
@@ -95,31 +95,49 @@ def get_tensor(graph, tfobj_or_name):
         'cannot locate tensor {} in current graph'.format(_tensor_name)
     return tnsr
 
-def as_tensor_name(name):
```

Comment:
There are multiple ways to get tensor/operation names in this utility module. 
We should definitely consolidate the functions `as_tensor_name` and `tensor_name`. 
Right now, the semantic of the two functions are:
1. `as_tensor_name`: return the tensor's name of the input (without necessarily check if it exists in any graph). 
2. `tensor_name`: return the tensor's name only if the input exists in the given graph.